### DISCUSSION TITLE
Keep floating element open on click?

### DISCUSSION THREAD

**Discussion by rob-johansen** [floating-ui/floating-ui#2745] [React] [answered]
Is there a way to keep a floating element open when it is clicked? Our designs call for items on a floating element to have space between their clickable regions, but when I inadvertently click between two items, the floating element closes. Here's a short video showing what I'd like to avoid:

https://github.com/floating-ui/floating-ui/assets/1420644/c1caae2c-0773-460c-93d7-2893f40b74ce

**Comment by atomiks** [comment, marked as answer]
Code reproduction? This seems like an issue with your own code as this doesn't happen with any of the library code: https://codesandbox.io/p/sandbox/pensive-silence-7rdnzw?file=%2Fsrc%2FDropdownMenu.tsx%3A229%2C15

**Reply by rob-johansen** [reply]
You're absolutely right—it was an issue in my code! 🤦‍♂️

Thanks for sharing the excellent codesandbox example.

**Comment by sarat1669** [comment]
How can we achieve this with vanilla js?

**Reply by sarat1669** [reply]
Ignore this. I can keep track of the state and show or hide accordingly.
